Tone/signal/Pow.ts

```diff
@@ -0,0 +1,85 @@
+import { Signal, SignalOptions } from "./Signal";
+import { WaveShaper, WaveShaperMappingFn } from "./WaveShaper";
+import { optionsFromArguments } from "Tone";
+
+/**
+ *  @class Pow applies an exponent to the incoming signal. The incoming signal
+ *         must be AudioRange.
+ *
+ *  @constructor
+ *  @param exp The exponent to apply to the incoming signal, must be at least 2. 
+ *  @example
+ * var pow = new Pow(2);
+ * var sig = new Signal(0.5).connect(pow);
+ * //output of pow is 0.25. 
+ */
+export class Pow extends Signal<number> {
```

Comment:
This Should extend [SignalOperator](https://github.com/Tonejs/Tone.js/blob/typescript/Tone/signal/SignalOperator.ts)

and pass in PowOptions as the generic argument to SignalOperator. This will let the get/set functions know what the defaults interface looks like

Reply:
This makes a lot more sense now, I was going towards that route, I basically forgot I could just extend ToneAudioNodeOptions to include the value for that class!